Dart/src/com/jetbrains/lang/dart/ide/generation/CreateConstructorFix.java

```diff
@@ -44,10 +44,9 @@ protected Template buildFunctionsText(TemplateManager templateManager, Set<DartC
         template.addTextSegment(",");
       }
     }
-    template.addTextSegment(")");
-    template.addTextSegment("{\n");
+    template.addTextSegment(");");
```

Comment:
With no body- yes.  Since the generator doesn't put any statements into the body, this is the natural choice.

Also, it is less keystrokes for users to add the braces, than to remove them.